docs-chef-io/content/inspec/resources/podman.md

```diff
@@ -0,0 +1,176 @@
++++
+title = "podman resource"
+draft = false
+gh_repo = "inspec"
+platform = "unix"
+
+[menu]
+  [menu.inspec]
+    title = "podman"
+    identifier = "inspec/resources/os/podman.md podman resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `podman` Chef InSpec audit resource to test the configuration data for the Podman resources.
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `podman` resource block declares allows you to write test for many `containers`:
+
+    describe podman.containers do
+      its('ids') { should include "591270d8d80d26671fd6ed622f367fbe19004d16e3b519c292313feb5f22e7f7" }
+      its('images) { should include "docker.io/library/ubuntu:latest" }
+    end
+
```

Comment:
also, I felt where you are providing code snippets here and above, the sentence is ending abruptly. Hence introduced, 'for example', and on such occurance throughout this resource writeup. Please check and cofirm on this too.